www/outsystems-secure-sqlite-init.js

```diff
@@ -38,13 +38,15 @@ function acquireLsk(successCallback, errorCallback) {
                     function (value) {
                         lskCache = value;
                         console.log("Got Local Storage key");
+                        OutSystemsNative.Logger.logWarning("Got Local Storage key", "SQLiteBundle");
                         successCallback(lskCache);
                     },
                     function (error) {
                         // If there's no key yet, generate a new one and store it
                         var newKey = generateKey();
                         lskCache = undefined;
                         console.log("Setting new Local Storage key");
+                        OutSystemsNative.Logger.logError("Setting new Local Storage key", "SQLiteBundle");
```

Comment:
We could log the key hash like we did in the other plugin, it may be easier to correlate with the other logs